pkg/util/jobutil/job_utils.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package jobutil
```

Comment:
I don't know that this warrants its own package. `jobs` already depends on all of this stuff. Seems fine to put this file in that package.